style.css

```
@@ -446,11 +446,13 @@ svg#main-chart rect[tabindex="0"]:focus {
     pointer-events: none;
     background: var(--abyss);
     border: 1px solid var(--border);
-    padding: 1.75rem;
+    padding: 1.25rem;
     border-radius: var(--radius-md);
     z-index: 100;
     min-width: 340px;
     max-width: min(340px, 90vw);
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)